.github/workflows/gradle.yml

```diff
@@ -275,6 +275,74 @@ jobs:
           SLACK_TITLE: "Build failure"
           SLACK_FOOTER: ""
 
+      - name: Slack Notification - Success
+        if: success() && github.ref == 'refs/heads/master'
+        uses: rtCamp/action-slack-notify@master
+        env:
+          SLACK_WEBHOOK: ${{ secrets.BUILD_SLACK_WEBHOOK }}
+          SLACK_USERNAME: Buildbot
+          SLACK_TITLE: "Build Success"
+          SLACK_FOOTER: ""
+  octavia-cli-build:
```

Comment:
@davinchia should these run on aws instances like the other builds?

Reply:
most of the work done is limited to the octavia module. only exception is format, but that isn't much faster on the big instances.

the current build is 5 mins, which I think is okay!

Reply:
When we discussed this with @davinchia we thought the job was light enough to run on the default ubuntu instance and not depend on the EC2 instance availability. But, in a second thought, it might be better to have a build context parity and use the EC2 instance for this build.